notebooks/Notebook3.md

```diff
@@ -18,17 +31,94 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+I spent a solid hour trying to figure out why my parser wasn't working. It kept
+telling me that it was finding a whitespace instead of end of input, and I was
+very confused. However, after a while of struggling, I figured out that it was
+because I accidentally deleted the space from my whitespace rule; see:
+
+	
+_ "whitespace"
+    = [ \\n\\r]*
+
+Instead of [ \\n\\r], I had done [\\n\\r] which was a mistake. 
+
+Once I fixed that, I could continue finishing up my parser. Last week, I had
+worked on just creating the grammar itself, but did not manipulate the output.
+This time, I worked on connecting my grammar to VexFlow, which is what will
+generate the music sheet.
+
+However, [VexFlow](https://www.vexflow.com/) was super annoying to work with.
```

Comment:
I looked a bit into vexflow, to see if It had an easy way to customize the lines in a clef, but I don't think it does. In vextab you're able to say `tabstave clef=percussion` to add a percussion symbol to the sheet music, but I don't believe more is possible in either vextab or vexflow.